docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -59,40 +59,19 @@ type: docs
 
 </ul>
 
```

Comment:
we should add this section 

https://deploy-preview-15391--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/troubleshoot/universe-issues/#configure-storage-class-volume-binding

to the "prepare nodes" page instead of in the troubleshooting section.

Reply:
Moved section